examples/charts/steps.py

```diff
@@ -1,9 +1,6 @@
 from collections import OrderedDict
 
-import numpy as np
-import pandas as pd
-from bokeh.charts import Chart, Step, Line
-from bokeh.plotting import show,output_file
+from bokeh.charts import Step, show, output_file
```

Comment:
I think I have a [no strong] preference to be able to do: 

```Python
from bokeh.plotting import show, figure
# or
from bokeh.charts import Line, show
# without have to always 
from bokeh.state import show
```
even if all `show` are the same method. Repeating is not a good pattern but in this case I see value in having it. (maybe I'm too lazy? :stuck_out_tongue_closed_eyes: )